lib/src/wiggle_abi/req_impl.rs

```diff
@@ -95,9 +95,9 @@ impl FastlyHttpReq for Session {
     }
 
     #[allow(unused_variables)] // FIXME JDC 2023-06-18: Remove this directive once implemented.
-    fn downstream_client_h2_fingerprint<'a>(
```

Comment:
The changes in this file don't look to be related to the description of this pull-request - could we have these in a different pull-request please? 🤞